mono/metadata/threadpool-worker-default.c

```diff
@@ -387,7 +387,7 @@ worker_park (void)
 	gboolean timeout = FALSE;
 	gboolean interrupted = FALSE;
 
-	mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_THREADPOOL, "[%p] worker parking", mono_native_thread_id_get ());
+	mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_THREADPOOL, "[%p] worker parking", (void *) mono_native_thread_id_get ());
```

Comment:
we usually use `gpointer` instead of `void *`, but in this case it's an ID and there is even a macro to safely cast it to an unsigned int: `MONO_NATIVE_THREAD_ID_TO_UINT`. @luhenry do you have an opinion on this?